src/SDKs/AnalysisServices/Management.Analysis/Generated/AnalysisServicesManagementClient.cs

```diff
@@ -255,7 +247,7 @@ public AnalysisServicesManagementClient(System.Uri baseUri, ServiceClientCredent
         /// <exception cref="System.ArgumentNullException">
         /// Thrown when a required parameter is null
         /// </exception>
-        public AnalysisServicesManagementClient(System.Uri baseUri, ServiceClientCredentials credentials, HttpClientHandler rootHandler, params DelegatingHandler[] handlers) : this(rootHandler, handlers)
+        public AnalysisServicesManagementClient(System.Uri baseUri, Microsoft.Rest.ServiceClientCredentials credentials, System.Net.Http.HttpClientHandler rootHandler, params System.Net.Http.DelegatingHandler[] handlers) : this(rootHandler, handlers)
```

Comment:
@michael-falk seems you are using an older version of autoRest.
Can you use the existing generate.cmd and if you need any info, use generate.cmd /? and you will get more information.